draft-ietf-quic-qpack.md

```diff
@@ -120,6 +120,12 @@ which has a fixed index over time.  Its entries are defined in Appendix A of
 {{!RFC7541}}. Note that because HPACK did not use zero-based references, there
 is no value at index zero of the static table.
 
+A decoder that encounters an invalid static table index on a request stream or
+push stream MUST treat this as an error of type
```

Comment:
As I am proposing to allow peers to signal a connection error instead of a stream error, I am not specifying the kind of error here and elsewhere where stream error would be sufficient.

Reply:
That's one way to deal with it.  In H2 and HQ, I think we've been leaning on the text that an implementation can always "upgrade" a stream error to a connection error at its discretion.  So even if we say stream, there's an implied choice.

Reply:
I think it would be more clear to use stream error here, and if an implementation wants to make it a connection error they can.

Reply:
Would it be better to have this text under the Absolute and Relative Indexing section, rather than here?